src/main/resources/application.yml

```diff
@@ -14,4 +14,21 @@ management:
       jmx:
         exposure.include: "*"
       web:
-        exposure.include: info, health
\ No newline at end of file
+        exposure.include: info, health
+app:
+    pdnd:
+      base-url: "\${PDND_BASE_URL:https://auth.uat.interop.pagopa.it}"
+      config:
+        audience: "\${PDND_ACCESS_TOKEN_AUDIENCE:auth.uat.interop.pagopa.it/client-assertion}"
+      anpr:
+        config:
+          client-id: "\${PAGOPA_PDND_CONFIGURATION_CLIENT_ID:890b7ca9-b402-4dce-9e8d-9a333d22d76d}"
+          kid: "\${PAGOPA_PDND_CONFIGURATION_KID:jxOpPRxM6oFcnnKtICqeW5l7fbxLr45IAsJ8Q9s-fK8}"
+          privateKey: "\${PDND_PRIVATE_KEY:}"
+          publicKey: "\${PDND_PUBLIC_KEY:}"
+        service-c003:
+          config:
+            purpose-id: "\${ANPR_PDND_CONFIGURATION_C003_PURPOSE_ID:87520bd5-207a-4616-85d9-10d7bb3e88b8}"
+        service-c030:
+          config:
```

Comment:
this is actually pretty confusing...
why don't explicit the configuration for each service? This way we will have just one PdndServiceIntegratedConfig to read per service

The fact to use the same values will be done defining just the most generic ENV required

app.pdnd.anpr.services:
   c003:
      client-id: "\${PDND_SERVICE_ANPR_C003_CLIENTID:\${PDND_SERVICE_ANPR_CLIENTID:\${PDND_SERVICE_CLIENTID:clientId}}}"
      kid: "\${PDND_SERVICE_ANPR_C003_KID:\${PDND_SERVICE_ANPR_KID:\${PDND_SERVICE_KID:kid}}}"
      privateKey: "\${PDND_SERVICE_ANPR_C003_PRIVATEKEY:\${PDND_SERVICE_ANPR_PRIVATEKEY:\${PDND_SERVICE_PRIVATEKEY:}}}"
      publicKey: "\${PDND_SERVICE_ANPR_C003_PUBLICKEY:\${PDND_SERVICE_ANPR_PUBLICKEY:\${PDND_SERVICE_PUBLICKEY:}}}"
      audience: "\${PDND_SERVICE_ANPR_C003_PUBLICKEY:anprc003audience}"